content/03.results-and-discussion.md

```diff
@@ -91,6 +93,7 @@ files and a reference FASTA file, this can be split into four main stages:
 4. Variant calling and consensus sequence generation: GATK
```

Comment:
Should this include pileupcaller too? Or do you classify that as not "Variant calling" since it just genotypes based on known variants?

Reply:
Ah yeah, please add.